app/components/web-components/undebate.jsx

```diff
@@ -2456,12 +2456,8 @@ class RASPUndebate extends React.Component {
                           <div style={{ textAlign: 'center' }}>
                             <span>Join and your recorded videos will be uploaded and shared</span>
                           </div>
-                          <div>
-                            <Join
-                              className={this.props.classes['join']}
-                              userInfo={{ name: this.state.name }}
-                              onChange={this.onUserLogin.bind(this)}
-                            ></Join>
+                          <div style={{ display: 'flex', justifyContent: 'center' }}>
+                            <AuthForm userInfo={{ name: this.state.name }} onChange={this.onUserLogin.bind(this)} />
```

Comment:
This is collecting firstName and lastName, if you add them to userInfo here they will get stored in the database.  You will also have to add firstName and lastName to app/models/user.js

I'm eager to see how you do it, I have not used react hooks but I want to do more.

Reply:
correction - it's in index.js handleLogin and handleSignUp that you need to add firstName and lastName to userInfo

Reply:
@ddfridley Will firstName and lastName be replacing the column "name" ?